src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesObsolete.TableExt.al

```diff
@@ -54,23 +54,23 @@ tableextension 308 NoSeriesObsolete extends "No. Series"
 
 #if not CLEAN24
 #pragma warning disable AL0432
-    [Obsolete('The function has been moved to codeunit NoSeriesManagement', '24.0')]
+    [Obsolete('The function has been moved to codeunit "No. Series Setup Impl."', '24.0')]
```

Comment:
General remark:
Sometimes you refer to "method", sometimes to "function".
The docs speak of "methods".
I personally prefer "procedure" for every method that was written in AL, but it would be helpful if there was some real Microsoft standard ;-)

Reply:
I'll try and stick to method from now on since that what the docs use...